README.md

```diff
@@ -0,0 +1,744 @@
+
+### Ingress Controller is needed
+This is unlike other types of controllers, which typically run as part of the kube-controller-manager binary, and which are typically started automatically as part of cluster creation.
+
+You can choose any Ingress controller:
+
+- Kubernetes currently supports and maintains GCE and nginx controllers.
+- F5 Networks provides support and maintenance for the F5 BIG-IP Controller for Kubernetes.
+- Kong offers community or commercial support and maintenance for the Kong Ingress Controller for Kubernetes
+
+### Why Nginx?
+- Nginx Ingress Controller officially supported by Kubernetes, as we already now.
+- it’s totally free :)
+- It’s the default option for minikube, which we will use to test Ingress behavior in Kubernetes.
+
+## Defaultbackend
+An Ingress with no rules sends all traffic to a single default backend. Traffic is routed to your default backend if none of the Hosts in your Ingress match the Host in the request header, and/or none of the paths match the URL of the request.
+
+### Nginx-Ingress and Defaultbackend
+```
+---
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ingress-nginx
+---
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: default-http-backend
+  labels:
+    app: default-http-backend
+  namespace: ingress-nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: default-http-backend
+  template:
+    metadata:
+      labels:
+        app: default-http-backend
+    spec:
+      terminationGracePeriodSeconds: 60
+      containers:
+      - name: default-http-backend
+        # Any image is permissible as long as:
+        # 1. It serves a 404 page at /
+        # 2. It serves 200 on a /healthz endpoint
+        image: gcr.io/google_containers/defaultbackend:1.4
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 30
+          timeoutSeconds: 5
+        ports:
+        - containerPort: 8080
+        resources:
+          limits:
+            cpu: 10m
+            memory: 20Mi
+          requests:
+            cpu: 10m
+            memory: 20Mi
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: default-http-backend
+  namespace: ingress-nginx
+  labels:
+    app: default-http-backend
+spec:
+  ports:
+  - port: 80
+    targetPort: 8080
+  selector:
+    app: default-http-backend
+---
+
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: nginx-configuration
+  namespace: ingress-nginx
+  labels:
+    app: ingress-nginx
+---
+
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: tcp-services
+  namespace: ingress-nginx
+---
+
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: udp-services
+  namespace: ingress-nginx
+---
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nginx-ingress-serviceaccount
+  namespace: ingress-nginx
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
why RBAC? do we enable it on clusteR? If so we need to explain these objects

Reply:
RBAC enabled by default in minikube since ... I don't even remember (v1.8?) we can describe people what it is to the audience in the previous sections of the workshop.

Reply:
Ok. Thank Ok I'lll add small RBAC introduction